includes/definitions/discovery/gaia.yaml

```diff
@@ -64,7 +65,7 @@ modules:
                     value: haState
                     states:
                         - { value: 1, descr: active , graph: 3, generic: 0 }
-                        - { value: 2, descr: standby, graph: 2, generic: 0 }
+                        - { value: 2, descr: standby, graph: 2, generic: 3 }
```

Comment:
I believe `generic` should be kept as `0`.
When looking at [Sensor-State-Support](https://docs.librenms.org/Developing/Sensor-State-Support/) a value off `3` is noted as `Unknown` which I don't think fit the description of `Standby`.

Reply:
I had'nt get it the generic code values idea, I was looking for a better view, but now make senses, I will adjust. Tks.